app/script/event/preprocessor/QuotedMessageMiddleware.js

```diff
@@ -55,17 +55,23 @@ z.event.preprocessor.QuotedMessageMiddleware = class QuotedMessageMiddleware {
 
     return this.eventService.loadEvent(event.conversation, quote.quoted_message_id).then(quotedMessage => {
       if (!quotedMessage) {
-        this.logger.warn('Quoted message not found');
-        // TODO add error metadata (to discuss with UI team)
+        this.logger.warn(`Quoted message with ID "${quote.quoted_message_id}" not found.`);
+        event.error = {
```

Comment:
Let's rather put that content in the `event.quote.error` instead of `event.error`.